File: HomeBuildApp/workflows/src/main/java/com/r3/developers/csdetemplate/homebuild/workflows/UpdateRequestWorkStateArgs.java
package com.r3.developers.csdetemplate.homebuild.workflows;

import java.util.UUID;

public class UpdateRequestWorkStateArgs {


    //you only need two variables here
    // id, and stauts.
    public UUID Id;
    public String requestStatus;

    public UpdateRequestWorkStateArgs() {
    }

    public UpdateRequestWorkStateArgs(UUID Id, String requestStatus) {
        this.Id = Id;
        this.requestStatus = requestStatus;
    }

    public UUID getId() {
        return Id;
    }

    public String getRequestStatus() {
        return requestStatus;
    }
}

File: HomebuildKotlin/workflows/src/main/kotlin/com/r3/developers/csdetemplate/homebuild_Workflows/UpdateRequestWorkStateArgs.kt
package com.r3.developers.csdetemplate.homebuild_Workflows

import java.util.*

class UpdateRequestWorkStateArgs {
    //you only need two variables here
    // id, and stauts.
    var id: UUID? = null
    var requestStatus: String? = null

    constructor()
    constructor(Id: UUID?, requestStatus: String?) {
        id = Id
        this.requestStatus = requestStatus
    }
}